src/NovelRT.Interop/NrtInteropErrorHandling.cpp

```diff
@@ -3,19 +3,25 @@
 
 #include <NovelRT.Interop/NrtErrorHandling.h>
 
-const char* const errMsgIsNullptr = "Unable to continue! A nullptr was passed when a ptr was expected.";
-const char* const errMsgIsNaN = "Unable to continue! Calculation resulted in an object that is not a number (NaN).";
-const char* const errMsgIsDivideByZero = "Unable to continue! Divide by zero was attempted.";
-const char* const errMsgIsNotSupportedInternal = "The current operation is not supported.";
-const char* const errMsgIsAlreadyDeletedOrRemoved =
-    "Unable to continue! The specific item has already been deleted or removed.";
-const char* const errMsgIsInitialisationFailure = "Unable to continue! Service could not be initialised.";
-const char* const errMsgIsFunctionNotFound = "Unable to continue! Failed to locate the specified managed function.";
-const char* const errMsgIsNotInitialised = "Unable to continue! The service is currently not initialised.";
-const char* const errMsgIsArgumentOutOfRange = "Unable to continue! The argument is out of range.";
-const char* const errMsgIsInvalidOperation = "Unable to continue! The current operation is invalid.";
-const char* const errMsgIsCharacterNotFound = "Unable to continue! The character can't be found.";
-const char* const errMsgIsOutOfMemory = "Insufficient memory to continue the execution of the program.";
+const char* const errMsgIsNullInstanceProvided = "Unable to continue. A null instance was provided when an instance was expected.";
+const char* const errMsgIsNaN = "Unable to continue. Calculation resulted in an object that is not a number (NaN).";
+const char* const errMsgIsDivideByZero = "Unable to continue. Divide by zero was attempted.";
+const char* const errMsgIsNotSupportedInternal = "Unable to continue. The current operation is not supported.";
+const char* const errMsgDupKeyGiven = "Unable to continue. Duplicate key given.";
+const char* const errMsgCompilationError = "Unable to continue. Compilation error.";
```

Comment:
What error is this for?

Reply:
Which are you calling out here? `errMsgCompilationError`? GH preview shows 4 different messages so just making sure the question is clear

Reply:
I clicked the last one, not sure why its done that. Yes I mean this.